modules/virt-networking-wizard-fields-web.adoc

```diff
@@ -16,7 +16,7 @@
 |Name for the Network Interface Card.
 
 |Model
-|Indicates the model of the Network Interface Card. Supported values are *e1000*, *e1000e*, *ne2k_pci*, *pcnet*, *rtl8139*, and *virtIO*.
+|Indicates the model of the Network Interface Card. Supported values are *e1000e* and *virtIO*.
```

Comment:
The way we indicate the virtio model in the specs is "virtio" (all non-capitals), not "virtIO".
It's not even written this way in the formal documentation (e.g. https://www.redhat.com/en/blog/introduction-virtio-networking-and-vhost-net, https://wiki.libvirt.org/page/Virtio), even when referring to the model name itself (let alone when using it as a spec value).

Reply:
Updated to match UI